deployment/packer/driver.py

```diff
@@ -0,0 +1,100 @@
+"""Helper functions to handle AMI creation with packer"""
+
+import boto3
+from boto import config
+import os
+from os.path import dirname
+import subprocess
+import re
+import logging
+
+LOGGER = logging.getLogger(__file__)
+LOGGER.addHandler(logging.StreamHandler())
+LOGGER.setLevel(logging.INFO)
+
+
+def get_recent_batch_ami(aws_profile):
+    """Finds the most recent Deep Learning AMI"""
+    filters = [
+        {'Name': 'architecture', 'Values': ['x86_64']},
+        {'Name': 'owner-id', 'Values': ['898082745236']},
+        {'Name': 'virtualization-type', 'Values': ['hvm']},
+    ]
+    sess = boto3.Session(profile_name=aws_profile)
+    ec2 = sess.client("ec2")
+    images = filter(
+        lambda x: re.search("Deep Learning AMI Amazon Linux .*", x['Name']),
+        ec2.describe_images(Filters=filters)['Images'])
+    images.sort(key=lambda x: x["CreationDate"])
+
+    LOGGER.info("Found AMI: %s", images[-1]["ImageId"])
+
+    return images[-1]["ImageId"]
+
+
+def get_project_root():
+    return dirname(dirname(dirname(os.path.realpath(__file__))))
+
+
+def get_git_sha():
+    """Function that executes Git to determine the current SHA"""
+    git_command = ['git',
```

Comment:
This should probably be `git rev-parse --short HEAD`.  I know this command exists in other places, but more recently we've been using `rev-parse`.